docs/frameworks/sveltekit.md

```diff
@@ -74,9 +53,58 @@ When `SvelteKit` fix the issue, we'll remove the internal `SvelteKitAdapterPlugi
 
 Some of the above options will be excluded if you already provide them or if you provide options where there may be a conflict between them: you can view the source code of the [SvelteKit PWA configuration module](https://github.com/antfu/vite-plugin-pwa/tree/main/src/integrations/sveltekit/config.ts) to verify that there are no conflicts.
 
-## Prompt for update
+## SvelteKit Pages
+
+If you want your application can work in offline, you should remove `hydrate: false` from all your pages, this will prevent to inject the layout and so will not work in offline.
+
+### Auto Update
 
-Since `SvelteKit` uses `SSR / SSG`, we need to add the `ReloadPrompt` component using `dynamic import`. `vite-plugin-pwa` plugin will only register the service worker on build, it is aligned with the current behavior of [SvelteKit service worker module](https://kit.svelte.dev/docs#modules-$service-worker).
+Since `SvelteKit` uses `SSR / SSG`, we need to call the `vite-plugin-pwa` virtual module using `dynamic import`.
+
+The best place to include the virtual call will be in main layout of the application (you should register it in any layout):
+
+::: details src/routes/__layout.svelte
+```html
+<script>
+  import { onMount } from 'svelte'
+  import { browser, dev } from '$app/env'
+
+  onMount(async () => {
+    if (!dev && browser) {
+      const { registerSW } = await import('virtual:pwa-register')
+      registerSW({
+        immediate: true,
+        onRegistered(r) {
+          // uncomment following code if you want check for updates
+          // r && setInterval(() => {
+          //    console.log('Checking for sw update')
+          //    r.update()
+          // }, 20000 /* 20s for testing purposes */)
+          console.log(`SW Registered: ${r}`)
+        },
+        onRegisterError(error) {
+          console.log('SW registration error', error)
+        }
+      })
+    }
+  })
+</script>
+
+<svelte:head>
+  {#if (!dev && browser)}
+    <link rel="manifest" href="/manifest.webmanifest">
+  {/if}
+</svelte:head>
+
+<main>
+  <slot />
+</main>
+```
+:::
+
+### Prompt for update
+
+Since `SvelteKit` uses `SSR / SSG`, we need to add the `ReloadPrompt` component using `dynamic import`. `vite-plugin-pwa` plugin will only register the service worker on build (check the [Development section](/guide/development)), it is aligned with the current behavior of [SvelteKit service worker module](https://kit.svelte.dev/docs#modules-$service-worker).
```

Comment:
```suggestion
Since `SvelteKit` uses SSR / SSG, we need to add the `ReloadPrompt` component using a dynamic `import`. `vite-plugin-pwa` plugin will only register the service worker on build (check the [Development section](/guide/development)), it is aligned with the current behavior of [SvelteKit service worker module](https://kit.svelte.dev/docs#modules-$service-worker).
```